### DISCUSSION TITLE
Communication of breaking changes...

### DISCUSSION THREAD

**Discussion by AmyBernhoft** [actions/virtual-environments#6009] [General questions] [answered]
Related to breaking change: https://github.com/actions/virtual-environments/issues/5930

Is there some sort of communication channel I should join to get notified of changes like this?

**Comment by mikhailkoliada** [comment, marked as answer]
@AmyBernhoft hello!

We do not provide any other mean but repository announcements and blog posts/twitter reposts about really dramatic changes
